code/controllers/subsystem/job.dm

```diff
@@ -133,6 +131,8 @@ SUBSYSTEM_DEF(job)
 
 	return 1
 
+/datum/controller/subsystem/job/proc/is_job_empty(rank)
+	return GetJob(rank).current_positions == 0
```

Comment:
I am just curious. Can we do this syntax now? Only 514 couldn't do that, right?

Reply:
We could always do this syntax, it was Dreamchecker that would reject it if you didn't specify a RETURN_TYPE